extensions/blocks/podcast-player/style.scss

```diff
@@ -255,6 +255,7 @@ $player-background: transparent;
 			width: $track-status-icon-size;
 			height: $track-status-icon-size;
 			margin-top: ($editor-font-size * $editor-line-height - $track-status-icon-size) / 2; // center vertically
+			fill: inherit;
```

Comment:
This is a bit out of scope. Just noticed that if the fill is not inherited explicitly, some themes (e.g. TwentyNineteen) can override the svg fill.